### PR TITLE
Ft 2 create user account #164682579

### PR DESCRIPTION
**Pull request by kyakusahmed**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,50 @@
...
+```
+Run the app
+```
+$ python run.py
+```
+
+
+### How to run the Tests:
+
+ open the terminal,activate virtual enviroment in the sendIT directory  and enter:
```
Comment (mendozabree) [inline README.md:38]:
```suggestion
 open the terminal,activate virtual enviroment in the directory  and enter:
```
Change to appropriate app

File: app/views.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+@app.route('/', methods=['GET'])
+def index():
+    """opening route."""
+    return jsonify({
+        "status": 200, 'message': 'welcome to Epic Email.'
+        }), 200
+
+
+@app.route('/api/v1/users', methods=['POST'])
```
Comment (mendozabree) [inline app/views.py:23]: This is not the endpoint for `signup` look through the doc and change it

File: app/views.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        data['password']
+    )
+    new_user = [{
+        "user_id": user['id'],
+        "message": "user registration successful"
+        }]
+    return jsonify({"data": new_user, "status": 201}), 201
+
+
+@app.route('/api/v1/users/login', methods=['POST'])
```
Comment (mendozabree) [inline app/views.py:50]: This is not the endpoint for `login` look through the doc and change it

File: app/views.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    check_user = emails.search_user_by_email(
+        data['email'].strip(), data['password'])
+    if not check_user:
+        return jsonify({"status": 200, "message": "register_first"}), 200
+    access_token = create_access_token(identity=check_user)
+    return jsonify({
+        'message': "Login successful", 'access_token': access_token}), 200
+
+
+@app.route('/api/v1/emails/user/received', methods=['GET'])
```
Comment (mendozabree) [inline app/views.py:71]: This is not the endpoint for this look through the doc and change it

**Review by mendozabree — CHANGES_REQUESTED** [review]
@kyakusahmed I like that you have made progress.
Please look through the comments I have given
Also please correct the pep8 errors, to see the errors you will have to check the link I posted in the channel.

File: app/views.py (hunk 4 above)
Comment (kyakusahmed) [reply app/views.py:71]: made changes

File: app/views.py (hunk 3 above)
Comment (kyakusahmed) [reply app/views.py:50]: made changes

File: app/views.py (hunk 2 above)
Comment (kyakusahmed) [reply app/views.py:23]: made changes

File: README.md (hunk 1 above)
Comment (kyakusahmed) [reply README.md:38]: made changes too

**Pull request merged**